src/dropdown/list/dropdown-list.component.ts

```diff
@@ -519,4 +521,11 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 		// this subscription will auto unsubscribe because of the `first()` pipe
 		(this._itemsReady || of(true)).pipe(first()).subscribe(subcription);
 	}
+
+	reorderSelected(moveFocus = false): void {
+		this.displayItems = [...this.getSelected(), ...this.getListItems().filter(item => !item.selected)];
+		if (moveFocus) {
+			setTimeout(() => this.getCurrentElement().focus(), 1);
```

Comment:
We should be able to elide the explicit timeout and have it fire on the next tick ... unless this is to account for focus delays or something?

```suggestion
			setTimeout(() => this.getCurrentElement().focus());
```